defs.bzl

```diff
@@ -34,13 +34,17 @@ def _pip_repository_impl(rctx):
         "@%s" % rctx.attr.name,
     ]
 
+    if rctx.attr.debug:
+        args.append("--debug")
```

Comment:
I'd prefer if this referenced pip and the actual option name, i.e `--pip-verbose`. Right now, it is ambiguous and implies debug settings for rules_python_external as a whole.

Reply:
Sure, will do.

Reply:
Fixed.